features/payment/managing_payment_methods/sorting_payment_methods_by_position.feature

```diff
@@ -0,0 +1,31 @@
+@managing_payment_methods
+Feature: Sorting listed payment methods by position
+    In order to change the order by which payment methods are displayed
+    As an Administrator
+    I want to sort payment methods by their positions
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store allows paying with "Paypal Express Checkout" at position 0
+        And the store allows paying with "Offline" at position 1
+        And the store allows paying with "Cash on Delivery" at position 2
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Payment methods are sorted by position in ascending order by default
+        When I am browsing payment methods
```

Comment:
`When I browse payment methods`
or
`Given I am browsing payment methods`
in my opinion